app/spdb/build.sbt

```diff
@@ -512,6 +516,7 @@ def gnagsodb(version: Version) = AppConfig(
     "-Dedu.gemini.site=north",
     "-Dcron.archive.edu.gemini.dbTools.html.ftpHost=gsagn.hi.gemini.edu",
     "-Dcron.odbMail.SITE_SMTP_SERVER=smtp.hi.gemini.edu",
+    "-Djava.rmi.server.hostname=gnauxodb",
```

Comment:
Yes... this is still the empty ODB we use for PIT AGS estimates. We haven't yet simplified this down, but will aim to do so for the PIT 2018A release.